lib/src/rules/avoid_annotating_with_dynamic.dart

```diff
@@ -81,7 +81,7 @@ class _Visitor extends SimpleAstVisitor<void> {
   }
 
   void _checkNode(NormalFormalParameter node, TypeAnnotation? type) {
-    if (type is NamedType && type.name.name == 'dynamic') {
+    if (type is NamedType && type.name2.lexeme == 'dynamic') {
```

Comment:
This is of course semantically the same as it was, but why now `type.type is DynamicType`?

Reply:
Yes, please!

Reply:
Done